src/neo_usboard_v2_node.cpp

```diff
@@ -92,15 +137,14 @@ int main(int argc, char** argv)
 	// initialize VNX
 	vnx::init("neo_usboard_v2_node", 0, nullptr);
 
-	ros::NodeHandle nh;
 	ros::NodeHandle nh_private("~");
 
 	std::string can_device;
 	std::string serial_port;
 	int can_baud_rate = 0;
 	double update_rate = 0;
 
-	nh_private.param<std::string>("can_device", can_device, "");
+	nh_private.param<std::string>("can_device", can_device, "can0");
```

Comment:
If you do that it will try to connect on the can bus by default, ignoring the serial_port parameter, that's why I left it blank here.